### PR TITLE
Session spent tokens

### PR DESCRIPTION
**Pull request by Waldz**
Updates: #1958

### CODE REVIEW

File: consumer/session/session_storage.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ const sessionStorageBucketName = "session-history"
 
 // StatsRetriever can fetch current session stats
 type StatsRetriever interface {
-	Retrieve() connection.Statistics
+	GetSessionStatistics() connection.Statistics
```
[the remark this answers is not in the document]
Comment (Waldz) [reply consumer/session/session_storage.go:32]: it's store not just about data statistics, but rather about current session: data stats, duration, payment stats, etc.

File: cmd/di.go [hunk 2]
```diff
@@ -383,22 +383,24 @@ func (di *Dependencies) bootstrapStorage(path string) error {
...
 
 func (di *Dependencies) subscribeEventConsumers() error {
-	// state events
+	// Consumer current session store
 	err := di.EventBus.Subscribe(connection.AppTopicConsumerSession, di.StatisticsTracker.ConsumeSessionEvent)
 	if err != nil {
 		return err
 	}
-	err = di.EventBus.Subscribe(connection.AppTopicConsumerSession, di.StatisticsReporter.ConsumeSessionEvent)
+	err = di.EventBus.Subscribe(connection.AppTopicConsumerStatistics, di.StatisticsTracker.ConsumeStatisticsEvent)
```
[the remark this answers is not in the document]
Comment (Waldz) [reply cmd/di.go:397]: I known. I'm reworking this in upcoming PR

File: consumer/session/session_storage.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (tadaskay) [reply consumer/session/session_storage.go:32]: Yes, but the package already indicates that, like I said.

[the remark this answers is not in the document]
Comment (Waldz) [reply consumer/session/session_storage.go]: oh. Improved naming

**Pull request merged**